internal/configs/version2/nginx.transportserver.tmpl

```diff
@@ -10,6 +10,10 @@ upstream {{ $u.Name }} {
     {{- range $s := $u.Servers }}
     server {{ $s.Address }} max_fails={{ $s.MaxFails }} fail_timeout={{ $s.FailTimeout }} max_conns={{ $s.MaxConnections }};
     {{- end }}
+
+    {{- range $b := $u.BackupServers }}
+    server {{ $b.Address }} backup;
+    {{- end }}
```

Comment:
Do we want to remove this from the OSS template now?

Reply:
done